install/kubernetes/helm/Istio/Chart.yaml

```diff
@@ -0,0 +1,13 @@
+name: Istio
+version: 0.1.0-1
+description: Helm chart for complete Istio deployment
+keywords:
+  - Istio
+  - auth
+  - initializer
+  - mixer
+  - pilot
+sources:
+  - http://github.com/istio/istio
+engine: gotpl
+#icon:  
```

Comment:
Should be able to use the icon from the mailing list.

Reply:
Would you useful to provide the link along with your comment.

Reply:
sure:
https://istio.io/favicons/android-192x192.png

Or from the team drive:
https://drive.google.com/drive/folders/0B7huSKYaiUN5czRkcU9SbUwtbDA

Reply:
done